Resources/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "fos-router",
+  "version": "1.6.0",
+  "description": "A pretty nice way to use the routes generated by the FOSJsRoutingBundle in your JavaScript.",
+  "keywords": [
+    "router",
+    "symfony"
+  ],
+  "license": "MIT",
+  "author": {
+    "name": "FriendsOfSymfony Community",
+    "url": "https://github.com/friendsofsymfony/FOSJsRoutingBundle/contributors"
+  },
+  "contributors": [
+    {
+      "name": "William Durand",
+      "email": "will+git@drnd.me"
+    },
+    {
+      "name": "Bruno Sampaio",
+      "email": "bens.sampaio@gmail.com"
+    }
+  ],
+  "files": [
+    "js/router.js",
+    "public/js/router.js",
```

Comment:
why do we include both files ?

Reply:
You're correct, shouldn't be including the first. Now updated.